qiskit/circuit/__init__.py

```diff
@@ -1047,6 +1047,23 @@ def __array__(self, dtype=None, copy=None):
 .. autofunction:: random_circuit
 .. currentmodule:: qiskit.circuit
 
+Apply Pauli twirling to a circuit
+---------------------------------
+
+There are two primary types of noise when executing quantum circuits. The first is stochastic,
+or incoherent, noise that is mainly due to the unwanted interaction between the quantum processor
+and the external environment in which it resides. The second is known as coherent error, and these
+errors arise due to imperfect control of a quantum system. This can be unwanted terms in a system
+Hamiltonian, i.e. incorrect unitary evolution, or errors from incorrect temporal control of the
+quantum system, which includes things like incorrect pulse-shapes for gates.
+
+Pauli-twirling is a quantum error suppression technique that uses randomization to shape coherent
```

Comment:
Minor nitpick to be consistent with other spellings used here 🙂 
```suggestion
Pauli twirling is a quantum error suppression technique that uses randomization to shape coherent
```